### PR TITLE
WPF - IME Transparent Text Background

### PR DESCRIPTION
**Pull request by nyjin**
**Fixes:** [issue-number] 
Issue #1262

**Summary:** IME text background color has been changed.
   - The background color of IME text has been changed to transparent.
      
**How Has This Been Tested?**  

tested using CefSharp.MinimalExample project

**Screenshots (if appropriate):**
before
asis
after
tobe

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Updated documentation

**Checklist:**

- [x] Tested the code(if applicable)
- [ ] Commented my code
- [ ] Changed the documentation(if applicable)
- [ ] New files have a license disclaimer
- [ ] The formatting is consistent with the project (project supports .editorconfig)

### CODE REVIEW

**Comment by amaitland** [comment]
Please ask on #1262 if others using `IME` are happy with your proposed change.

**Comment by amaitland** [comment]
I think we'll make this configurable, we if change the default to transparent, anyone who wishes to revert to white will have the option.

**Comment by nyjin** [comment]
I added static methods on imehandler for set background color or underline color through wpfimekeyboardhandler constructor.
Please review my request.

File: CefSharp.Wpf.Example/Views/BrowserTabView.xaml.cs [hunk 1]
```diff
@@ -31,6 +33,8 @@ public BrowserTabView()
 
             //browser.BrowserSettings.BackgroundColor = Cef.ColorSetARGB(0, 255, 255, 255);
 
+            browser.WpfKeyboardHandler = new WpfImeKeyboardHandler(browser, Colors.Transparent);
+
```
Comment (amaitland) [inline CefSharp.Wpf.Example/Views/BrowserTabView.xaml.cs:37]: Please at a minimum comment this out, add a comment saying uncomment to enable experimental IME handling.

File: CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs [hunk 2]
```diff
@@ -35,8 +36,19 @@ public class WpfImeKeyboardHandler : WpfKeyboardHandler
         /// Constructor.
         /// </summary>
         /// <param name="owner">The owner.</param>
-        public WpfImeKeyboardHandler(ChromiumWebBrowser owner) : base(owner)
+        /// <param name="backgroundColor">SkColor value for background.</param>
+        /// <param name="underlineColor">SkColor value for underline.</param>
+        public WpfImeKeyboardHandler(ChromiumWebBrowser owner, Color? backgroundColor = null, Color? underlineColor = null) : base(owner)
```
Comment (amaitland) [inline CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs:41]: Revert this please.

File: CefSharp.Wpf/Internals/IMEHandler.cs [hunk 3]
```diff
@@ -15,9 +16,19 @@ namespace CefSharp.Wpf.Internals
     public static class ImeHandler
     {
         // Black SkColor value for underline.
-        internal const uint ColorUNDERLINE = 0xFF000000;
+        internal static uint ColorUNDERLINE = 0xFF000000;
         // White SkColor value for background.
-        internal const uint ColorBKCOLOR = 0xFFFFFFFF;
+        internal static uint ColorBKCOLOR = 0xFFFFFFFF;
```
Comment (amaitland) [inline CefSharp.Wpf/Internals/IMEHandler.cs:21]: For simplicity we'll just make them public static. Your code change implies they can be set per WpfImeKeyboardHandler instance which isn't the case. 

http://cefsharp.github.io/api/83.4.x/html/M_CefSharp_Cef_ColorSetARGB.htm can be used to create the colour uint

File: CefSharp.Wpf.Example/Views/BrowserTabView.xaml.cs (hunk 1 above)
Comment (nyjin) [reply CefSharp.Wpf.Example/Views/BrowserTabView.xaml.cs:45]: Added comments. :)

File: CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs (hunk 2 above)
Comment (nyjin) [reply CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs]: Reverted.

File: CefSharp.Wpf/Internals/IMEHandler.cs (hunk 3 above)
Comment (nyjin) [reply CefSharp.Wpf/Internals/IMEHandler.cs]: I intended to set both the ime handler and color within a limited range, but I respect your opinion.
Fixed.

**Comment by amaitland** [comment]
Is the background transparent by default now? Happy to change now it's configurable.

**Pull request merged**